charts/fulcio/Chart.yaml

```diff
@@ -5,7 +5,7 @@ description: |
 
 type: application
 
-version: 1.0.0
+version: 2.0.0
```

Comment:
you bump to 2.0.0 there is any breaking change?
if yes can you write those down and maybe how users can upgrade existing deployment

thanks!